### PR TITLE
feat: more versatile errors on homepage

### LINKED ISSUE #71: More versatile error state message on the home page
* BTC/BTC, LTC/LTC, L-BTC/L-BTC, L-LTC/L-LTC :  Turn the bottom bar into red and display "Choose different asset " 

* BTC/LTC and LTC/BTC: Turn the bottom bar into grey and display "Coming soon"

The above two message will override the default "invalid amount" error message, i.e, Only check for "invalid amount" error state if the above two conditions are passed.

### PR DESCRIPTION
**Pull request by ImmanuelSegol**
closes #71

### CODE REVIEW

File: src/components/swaptab/index.js [hunk 1]
```diff
@@ -167,27 +169,48 @@ class SwapTab extends React.Component {
     return baseAmount <= MAX && baseAmount >= MIN;
   };
 
+  checkValidPair = (quote, base) => {
+    if (quote === base) {
+      this.setState({
+        error: true,
+        errorMessage: 'Choose different asset',
```
Comment (michael1011) [inline src/components/swaptab/index.js]: Either `Choose different assets` (plural) or `Choose a different asset` (add an `a`)

File: src/components/swaptab/index.js [hunk 2]
```diff
@@ -252,23 +283,23 @@ class SwapTab extends React.Component {
...
           </View>
         </View>
         <View className={classes.next}>
           <Controls
             text={'Start swap'}
-            error={error}
+            error={error || inputError}
             onPress={error ? () => {} : () => this.shouldSubmit()}
-            errorText={'Invalid amount'}
+            errorText={inputError ? 'Invalid amount' : errorMessage}
```
Comment (michael1011) [inline src/components/swaptab/index.js]: Do you think it is fine to have `Invalid amount` as default error? Would it be better to get rid of the `error` boolean and just have the string `errorText` that determines whether there is an error and sets the text of the button?

Comment (ImmanuelSegol) [reply src/components/swaptab/index.js]: @michael1011 My idea was that the input error would always override any other error.

Comment (michael1011) [reply src/components/swaptab/index.js]: Yeah that sounds reasonable.

**Pull request merged**